formats/service_manual_guide/publisher_v2/examples/with_change_history.json

```diff
@@ -0,0 +1,33 @@
+{
+  "publishing_app": "service-manual-publisher",
+  "rendering_app": "government-frontend",
+  "public_updated_at" : "2015-10-09T08:17:10+00:00",
+  "format": "service_manual_guide",
+  "locale": "en",
+  "details" : {
+    "body" : "<h2 id=\"what\">What it is, why it works and how to do it</h2>\n<p>Agile methodologies will help you and your team to build world-class, user-centred services quickly and affordably.</p>\n",
+    "related_discussion" : {
+      "title" : "Design community on hackpad",
+      "href" : "https://designpatterns.hackpad.com/"
+    },
+    "header_links" : [
+      { "title" : "What is it, why it works and how to do it", "href" : "#what-is-it-why-it-works-and-how-to-do-it" }
+    ],
+    "change_history": [
+      {
+        "public_timestamp": "2015-10-09T08:17:10+00:00",
+        "note": "This is a change",
+        "full_description": "This is why we made this change"
```

Comment:
It would be useful if the example used multiline text to demonstrate how this works.